### PR TITLE
[maintenance] Upgrades webserver's requirements

### PR DESCRIPTION
**Pull request by pcrespov**
## What do these changes do?

Upgrade strategy starts by upgrading all package requirements to their latest versions and testing them. This establishes constraints on base input requirements that are included in the webserver service's input requirements. These requirements are compiled to get latest upgrades. If fails, constraints are added either in the input requirements of the service or in one of more of the packages, depending on the nature of the failure.

Specifically, these are the changes:

- Review and upgrade test requirements of **packages**:
   - [x] postgres-database.
   - [x] servicelib
   - [x] simcore sdk
   - [x] s3wrapper
   - [x] pytest-simcore
- Upgrades **webserver's** dependencies:
   - Minor fixes in fixtures due to engineio. CANNOT upgrade due to issue reported in #1445 
   - Changes base image from alpine to slim: new libraries failed to compile. Updated as well booting scripts
- [x] Makefile cleanup
    - fixes ``make clean`` under different conditions
    - new ``make clean-ps`` to remove docker containers
    - silent pips
    - ``make reqs`` in webserver now include dependencies to ``_base.in`` packages

- **FIXES**: exec boot of webserver and director to grant graceful shutdown

## Related issue number

Part of #1412 to upgrade python

## How to test

```cmd
make devenv
cd services/web/server
make install-dev tests
```

## Checklist

- [x] Did you change any service's API? Then make sure to bundle document and upgrade version (``make openapi-specs``, ``git commit ...`` and then ``make version-*``)
- [x] Unit tests for the changes exist
- [x] Runs in the swarm
- [x] Documentation reflects the changes
- [x] New module? Add your github username to [.github/CODEOWNERS](.github/CODEOWNERS)

### CODE REVIEW

File: scripts/requirements.Makefile [hunk 1]
```diff
@@ -6,13 +6,16 @@
...
 .DEFAULT_GOAL := help
 
+UPGRADE_OPTION := $(if $(upgrade),--upgrade-package $(upgrade),--upgrade)
+
+
 objects = $(wildcard *.in)
 outputs := $(objects:.in=.txt)
 
-reqs: $(outputs) ## pip-compiles all requirements/*.in -> requirements/*.txt
+reqs: $(outputs) ## pip-compiles all requirements/*.in -> requirements/*.txt; make reqs upgrade=foo will only upgrade package foo
```
Comment (sanderegg) [inline scripts/requirements.Makefile:15]: very cool!

File: services/web/Dockerfile [hunk 2]
```diff
@@ -155,7 +155,9 @@ ENV SIMCORE_WEB_OUTDIR /devel/services/web/client
 
 WORKDIR /devel
 
-# NOTE: declaring VOLUMEs here makes troubles mounting 
+RUN chown -R scu:scu ${VIRTUAL_ENV}
```
Comment (sanderegg) [inline services/web/Dockerfile:158]: good one that one...

File: services/web/server/tests/unit/with_dbs/conftest.py [hunk 3]
```diff
@@ -241,22 +241,27 @@ def is_redis_responsive(host: str, port: str) -> bool:
...
+            headers.update({'Cookie': security_cookie})
+
+        await sio.connect(url, headers=headers)
         assert sio.sid
         clients.append(sio)
         return sio
 
     yield connect
+
     for sio in clients:
```
Comment (sanderegg) [inline services/web/server/tests/unit/with_dbs/conftest.py:259]: is this one different than the one in the pytest-simcore? if yes we should just remove it if possible.

Comment (pcrespov) [reply services/web/server/tests/unit/with_dbs/conftest.py:259]: is the same one. I simply did not dare changing it until we resolve the issue with socketio.

**Comment by pcrespov** [comment]
@sanderegg could you please look at the tests failing due to socketio and give some feedback? THX!

**Pull request merged**